PROJECT

```diff
@@ -16,6 +16,9 @@ resources:
   kind: EdgeDevice
   path: github.com/project-flotta/flotta-operator/api/v1alpha1
   version: v1alpha1
+  webhooks:
```

Comment:
although not part of this PR, can you add the same for EdgeDeployment below?

Reply:
done